client/components/jetpack/connection-health/index.tsx

```diff
@@ -42,27 +33,27 @@ export const JetpackConnectionHealthBanner = ( { siteId }: Props ) => {
 		return;
 	}
 
-	return (
-		<>
-			<TrackComponentView
-				eventName="calypso_jetpack_connection_health_issue_view"
-				eventProperties={ { type: 'default' } }
+	if ( jetpackConnectionHealth?.error === DNS_ERROR ) {
+		return (
+			<ErrorNotice
+				eventType="dns"
+				errorText={ translate(
+					"Your domain is not properly set up to point to your site. Reset your domain's A records in the Domains section to fix this."
+				) }
+				noticeActionHref={ `/domains/manage/${ siteSlug }` }
```

Comment:
Will there be some error UX on the domains page that guides the user through fixing their problem? If not, we should keep pointing them to the support doc.

Reply:
Good point. I've checked it more, and the list of domains doesn't show any problems so the user could get lost.

Even if we pointed the user to the domain configuration (e.g.,/domains/manage/SITE_SLUG/edit/DOMAIN), they would still don't know that they need to enable WPCOM DNS in the 'Name servers' tab. Also, we still wouldn't cover a case in which the user manages domain DNS externally and has A records pointing to another server.